docs/aerodromes/classd/Moorabbin.md

```diff
@@ -81,13 +116,88 @@ Aircraft who will transit Class G airspace on climb into CTA will be **cleared t
     **SGE**: "Cleared to leave and re-enter controlled airspace climbing to A050, Departures 129.4, SGE"
 
 ### Arrivals/Overfliers
-ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
-IFR aircraft will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
-VFR aircraft will not be coordinated, and will need to receive their airways clearance from MB ADC.
+
+- ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
+- **IFR arrivals** will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
+- **VFR arrivals** will not be coordinated, and will need to receive their airways clearance from MB ADC.
 
 !!! example
     <span class="hotline">**MDS** -> **MB ADC**</span>: "To the west, PLE, for the NDB-A"  
     <span class="hotline">**MB ADC** -> **MDS**</span>: "PLE, NDB-A"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
+
+## Helicopters
+
+### Circuits
+
+#### Day VFR Circuits
+Helicopter circuits are conducted from designated areas based on the active runway:
+
+<figure markdown>
+![Eastern Grass and Western Triangle](img/mbhelicircuitareas.png){ width="700" }
+<figcaption>Eastern Grass and Western Triangle</figcaption>
+</figure>
+
+- **RWY 17/35**: Circuits are conducted from the **Eastern Grass**.
+- **RWY 13/31**: Circuits are conducted from the **Western Triangle**.
+- Circuit operations are **parallel to the duty runway** and inside the fixed-wing circuit.
```

Comment:
I'm assuming based on the example below that these areas are outside ADC's jurisdiction. Would we be able to indicate as such, so that controllers don't issue takeoff/landing clearances?

Reply:
Western triangle and eastern grass are parts of the runway/active environment and include grass areas . Essentially they land in the unused area not used by current traffic. Similar to how at Essendon they have an eastern grass which is a portion of the aerodrome area for helicopters to train.

So it's part of the adc controller area. I'm doing a diagram based on vatsys. I have some Mac (Moorabbin corporation) materials I am converting to vatsys based diagrams

Western triangle is extended A to the boundary fence, up to the centreline of A4, then across to the edge of B and runs parallel B to the boundary fence
Eastern grass is basically everything east of 35R
I’ll put that in the doc too tho along with the diagram